From a45f3c1086c420c1cd08170d07f6bfc63a0f49cd Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 25 Nov 2014 13:44:54 -0500
Subject: [redhat] Add ppc64le build in makefile for rh-kabi target

Message-id: <a3db339c44d549219ae9957d826d0903ae0b34a4.1416921247.git.poros@redhat.com>
Patchwork-id: 100879
O-Subject: [PATCH RHEL7.1 2/3] [redhat] Add ppc64le build in makefile for rh-kabi target.
Bugzilla: 1149269
RH-Acked-by: Kyle McMartin <kmcmarti@redhat.com>

Patch adding kabi whitelist build for ppc64le.
    Also update sources-rh target for add Module.kabi_ppc64le into srpm

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile b/redhat/Makefile
index 0d7fead..ecdcce0 100644
--- a/redhat/Makefile
+++ b/redhat/Makefile
@@ -32,7 +32,7 @@ include Makefile.cross
 default: rh-help
 
 rh-kabi:
- @for KABIARCH in "ppc64" "x86_64" "s390x"; do \
+ @for KABIARCH in "ppc64" "ppc64le" "x86_64" "s390x"; do \
   $(REDHAT)/kabi/show-kabi -k $(REDHAT)/kabi/kabi-module/ -m \
    -a $$KABIARCH -r $(RHELMAJOR).$(RHELMINOR) > $(REDHAT)/kabi/Module.kabi_$$KABIARCH;\
   for i in {0..$(RHELMINOR)}; do \
@@ -117,7 +117,7 @@ sources-rh: $(TARBALL)
  @(if [ ! -f keys/extra_certificates ]; then \
   touch keys/extra_certificates; \
  fi)
- @cp cpupower.* x509.genkey keys/rhel*.x509 keys/extra_certificates Makefile.common kabi/check-kabi kabi/Module.kabi_{ppc64,s390x,x86_64} configs/kernel-*.config rhpkg/Makefile scripts/sign-modules keys/*.cer $(SOURCES)/
+ @cp cpupower.* x509.genkey keys/rhel*.x509 keys/extra_certificates Makefile.common kabi/check-kabi kabi/Module.kabi_{ppc64,ppc64le,s390x,x86_64} configs/kernel-*.config rhpkg/Makefile scripts/sign-modules keys/*.cer $(SOURCES)/
  @(cd kabi && tar cjvf $(SOURCES)/kernel-abi-whitelists.tar.bz2 kabi-rhel7* kabi-current)
 
 rh-sources: setup-source rh-configs rh-kabi sources-rh
-- 
1.7.1